kuma/wiki/jinja2/wiki/includes/kumascript_errors.html

```diff
@@ -17,7 +17,7 @@
   </button>
   <div class="kserrors-details hidden">
       <p>{% trans %}
-        Anyone with an MDN profile can edit a document to fix an error. Use the <a href="/docs/MDN/Kuma/Troubleshooting_KumaScript_errors">KumaScript troubleshooting guide</a> as a starting point.
+        Anyone with an MDN profile can edit a document to fix an error. Use the <a href="/docs/MDN/Contribute/Tools/KumaScript/Troubleshooting">KumaScript troubleshooting guide</a> as a starting point.
```

Comment:
Good eye for detail on this link. While we're updating the string, let's lift the URL out of the translation, like we do for the [copyright URL](https://github.com/mozilla/kuma/blob/master/jinja2/base.html#L304-L307)